### DISCUSSION TITLE
Week 4 Roundup

### DISCUSSION THREAD

**Discussion by DotNetDublin** [BlazingSaddlesSolutions/TimeKeeper#69] [General]
This week we updated the TimeKeeper application to read Categories and Clients from the database following the vertical slices approach provided by @jonhilt.

@AnkurSheel generously submitted a [pull request](https://github.com/BlazingSolutions/TimeKeeper/pull/65) to provide Docker support which we should aim to review this week.

A very high level [roadmap](https://github.com/BlazingSolutions/TimeKeeper/wiki/Roadmap) is now also available in the project Wiki.

As outlined in the roadmap I think the immediate task is to reach agreement on a wireframe for the Time Entry page. See this [User Story](https://github.com/BlazingSolutions/TimeKeeper/issues/13) issue for details.

All feedback welcome.

**Comment by jonhilt** [comment]
Good stuff, thanks as always @DotNetDublin :-)

I've reviewed and added my feedback to the Docker pull request. I did manage to get it running on my machine with a few small tweaks so just awaiting feedback (to my feedback!) from @AnkurSheel on that one.

Great idea with the roadmap. I'll try to make time to take a glance at the Time Entry page wireframes as well.
